tvm_linker/src/disasm.rs

```diff
@@ -2730,6 +2730,7 @@ mod tests {
     }
 
     #[test]
+    #[ignore]
```

Comment:
Why do you need this?

Reply:
That's my mistake, I've ignored this test when it was broken. I've returned it

Reply:
Thanks!